imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc

```diff
@@ -0,0 +1,156 @@
+        "kycUrl": "https://verify.test.immersve.com/kyc/session/hg40904ddkr43?code=r938j9g"
+      }
+    },
+  ]
+}
+```
+{% /code %}
+
+If KYC is required, the response to the spending prerequisites request will
+contain a URL, where the user must be redirected to to complete the KYC
+verification. If KYC is not required, is already in progress, or the maximum
+number of retries has been exceeded, the URL will not be returned and the status
+will indicate the problem. See [Spending Prerequisites endpoint
+reference](https://docs.immersve.com/api-reference/get-spending-prerequisites/)
+for more details on possible statuses.
+
+
+### 3. User goes through the KYC flow
+
+Upon being redirected to the Immersve KYC page, the user will go through the
+process that involves the collection of the user's address and the verification
+of their identity through document and biometry (facial scan) collection.
+
+At the end of the process they are presented with a screen indicating that their
+profile is under review. The user does not have to stay on the page to know
+the outcome, as it will be communicated via email. If they stay on the page, it
+will eventually inform about success or failure.
+
+Upon exiting the page, users are sent back to the **`redirectUrl`** provided
+initially. If no URL was provided, exiting leads to closing the browser tab.
+
+![KYC UI screenshot](@assets/kyc/kyc-ui.png)
+
+
+### 4. Confirmation of KYC Completion
+
+Once redirected, check the spending prerequisites again to
+ensure no further KYC actions are required. Once there is no KYC prerequisite
+returned, and given all other checks are completed, the cardholder will be
+allowed to create and use their Immersve card.
+
+**Example Request:**
+
+{% code %}
+```bash
+curl -X POST "https://test.immersve.com/api/spending-prerequisites" \
+-H "Authorization: Bearer $token" \
+--data '{
+  "cardProgramId": "'${card_program_id}'",
+  "fundingSourceId": "'${funding_source_id}'",
+  "spendableAmount": 100,
+  "spendableCurrency": "USD",
+  "kycType": "immersve-conducted",
+  "kycRedirectUrl: "https://app.example.io"
+  "kycRegion":"NZL"
+}'
+```
+{% /code %}
+
+**Example Response:**
+
+{% code %}
+```json
+{
+  "prerequisites": []
+}
+```
+{% /code %}
+##
+
+## Testing
+
+- **ID Verification (IDV) Without Real Documents**: To support testing and
+simplify the integration process in test environments, we use simulated
+documents. This allows testing the KYC verification workflow without putting real
+details into test environments. Below are the front and back images of a driver license that can
+be scanned with a phone directly:
+
+  - **Front of a drivers license**
+
+  ![mock-licence-front (1).jpg](@assets/kyc/mock-driver-licence-front.jpg)
+
+  - **Back of a drivers license**
+
+  ![mock-licence-back (1).jpg](@assets/kyc/mock-driver-licence-back.jpg)
+
+
+{% note %}
+Facial scan still needs to be done with the testing person's face, but no data will
```

Comment:
This looks weird as a callout on the bottom of the page. I think it should be added into the prior paragraph just before the sentence "Below are the front and back images".